airbyte-commons/src/main/java/io/airbyte/commons/io/LineGobbler.java

```diff
@@ -40,24 +43,39 @@
   private final static Logger LOGGER = LoggerFactory.getLogger(LineGobbler.class);
 
   public static void gobble(final InputStream is, final Consumer<String> consumer) {
+    gobble(is, consumer, "generic");
+  }
+
+  public static void gobble(final InputStream is, final Consumer<String> consumer, String caller) {
     final ExecutorService executor = Executors.newSingleThreadExecutor();
     final Map<String, String> mdc = MDC.getCopyOfContextMap();
-    executor.submit(new LineGobbler(is, consumer, executor, mdc));
+    var gobbler = new LineGobbler(is, consumer, executor, mdc, caller);
+    executor.submit(gobbler);
   }
 
   private final BufferedReader is;
   private final Consumer<String> consumer;
   private final ExecutorService executor;
   private final Map<String, String> mdc;
+  private final String caller;
```

Comment:
without the `caller` field, it was difficult to debug what gobbler was running into errors.